runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -634,5 +636,31 @@ private void resetEndpoints()
 	{
 		// Select the first tab (NORMAL hiscores)
 		tabGroup.select(tabGroup.getTab(0));
+
+		// Selects the DMM tab if the current world is DMM.
+		selectWorldEndpoint();
+	}
+
+	private void selectWorldEndpoint()
+	{
+		if (client != null)
+		{
+			EnumSet<WorldType> wTypes = client.getWorldType();
+
+			if (wTypes.contains(WorldType.DEADMAN_TOURNAMENT))
+			{
+				tabGroup.select(tabGroup.getTab(6));
```

Comment:
Using constants instead of arbitrary numbers would improve readability
`tabGroup.getTab(6)`

Reply:
I think this can use the enum ordinals? eg. `WorldType.DEADMAN_TOURNAMENT.ordinal()`

Reply:
I don't know about WorldType ordinals.. perhaps HiscoreEndpoint ordinals since they're added that way via HiscoreEndpoint.values()